docs/developer/getting-started.rst

```diff
@@ -107,3 +107,11 @@ To run the Python tests, run (in the ``python/`` directory):
 
   cd python
   PYTHONPATH=$PYTHONPATH:./install python3 -m pytest
+
+Building Documentation
+----------------------
+
+- Run ``python3 data/fetch_neutron_data.py``
+- cd to a where the docs should be built (e.g. ``mkdir -p build/docs && cd build/docs``)
+- Ensure you have the correct conda environment loaded (so that scipp is detected)
```

Comment:
Specify that this should be the environment with Mantid (for Linux) or Mantid should be otherwise available, or the user should disable fail on Nbsphinx Python errors (can't remember what the actual config option is called).